config/jobs/kubernetes-sigs/cluster-api-provider-gcp/cluster-api-provider-gcp-presubmits.yaml

```diff
@@ -0,0 +1,66 @@
+presubmits:
+  kubernetes-sigs/cluster-api-provider-gcp:
+  - name: pull-cluster-api-provider-gcp-build
+    agent: kubernetes
+    always_run: true
+    context: pull-cluster-api-provider-gcp-build
+    rerun_command: "/test pull-cluster-api-provider-gcp-build"
+    trigger: "/test( all| pull-cluster-api-provider-gcp-build)"
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/gcloud-in-go:v20171113-192bec25
+        args:	
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
```

Comment:
sigs.k8s.io, as this is not part of k8s.io... same as below

Reply:
Whoops. Sent you https://github.com/kubernetes/test-infra/pull/8762 to fix.